bokehjs/src/lib/models/widgets/tables/cell_formatters.ts

```diff
@@ -207,6 +207,35 @@ export class BooleanFormatter extends CellFormatter {
   }
 }
 
+export namespace CompositeFormatter {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = CellFormatter.Props & {
+    icon: p.Property<string> // XXX: enum
```

Comment:
This icon property should be replaced with something like:
```
    formatters: p.Property<Any[]> 
```

cc @mattpap is there a better alternative for instances at this point?

Reply:
@bryevdv It looks like I need to import `Any`. I am thinking this would be imported from `bokehjs/src/lib/core/properties.ts`, would that be correct?

Reply:
> `formatters: p.Property<Any[]>`

This is confusing value type and property type. If anything, it should use lower-case `any`, but the right type is `p.Property<CellFormatter[]>`. Then one will use `p.Array` in property definition.